[INFO][2023-06-01 23:24:07][__init__.py:29] - [visual_dl] init failed, error_info: ModuleNotFoundError("No module named 'torch'")
[INFO][2023-06-01 23:24:07][__init__.py:29] - [visual_dl] init failed, error_info: ModuleNotFoundError("No module named 'torch'")
[INFO][2023-06-01 23:24:08][chatgpt.py:183] - proxy no find, directly request to chatgpt instead
[INFO][2023-06-01 23:24:08][chatgpt.py:183] - proxy no find, directly request to chatgpt instead
[INFO][2023-06-01 23:24:08][victorinox.py:86] - use_tools=['python', 'meteo-weather', 'url-get', 'terminal'], params: {'debug': False, 'openai_api_key': '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', 'open_ai_api_base': 'https://api.openai.com/v1', 'proxy': '', 'request_timeout': 120, 'model_name': 'gpt-3.5-turbo', 'no_default': False, 'top_k_results': 3, 'news_api_key': '', 'bing_subscription_key': '', 'google_api_key': '', 'google_cse_id': '', 'searx_search_host': '', 'wolfram_alpha_appid': '', 'morning_news_api_key': '', 'cuda_device': 'cpu', 'think_depth': 3, 'arxiv_summary': True, 'morning_news_use_llm': False, 'llm_api_key': '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', 'llm_api_base_url': 'https://api.openai.com/v1'}
[INFO][2023-06-01 23:56:23][__init__.py:29] - [visual_dl] init failed, error_info: ModuleNotFoundError("No module named 'torch'")
[INFO][2023-06-01 23:56:24][__init__.py:29] - [visual_dl] init failed, error_info: ModuleNotFoundError("No module named 'torch'")
[INFO][2023-06-01 23:56:24][chatgpt.py:183] - proxy no find, directly request to chatgpt instead
[INFO][2023-06-01 23:56:24][chatgpt.py:183] - proxy no find, directly request to chatgpt instead
[INFO][2023-06-01 23:56:24][victorinox.py:86] - use_tools=['terminal', 'url-get', 'python', 'meteo-weather'], params: {'debug': False, 'openai_api_key': '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', 'open_ai_api_base': 'https://api.openai.com/v1', 'proxy': '', 'request_timeout': 120, 'model_name': 'gpt-3.5-turbo', 'no_default': False, 'top_k_results': 3, 'news_api_key': '', 'bing_subscription_key': '', 'google_api_key': '', 'google_cse_id': '', 'searx_search_host': '', 'wolfram_alpha_appid': '', 'morning_news_api_key': '', 'cuda_device': 'cpu', 'think_depth': 3, 'arxiv_summary': True, 'morning_news_use_llm': False, 'llm_api_key': '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', 'llm_api_base_url': 'https://api.openai.com/v1'}
[INFO][2023-06-01 23:58:34][__init__.py:29] - [visual_dl] init failed, error_info: ModuleNotFoundError("No module named 'torch'")
[INFO][2023-06-01 23:58:34][__init__.py:29] - [visual_dl] init failed, error_info: ModuleNotFoundError("No module named 'torch'")
[INFO][2023-06-01 23:58:34][chatgpt.py:183] - proxy no find, directly request to chatgpt instead
[INFO][2023-06-01 23:58:34][chatgpt.py:183] - proxy no find, directly request to chatgpt instead
[INFO][2023-06-01 23:58:34][victorinox.py:86] - use_tools=['url-get', 'meteo-weather', 'python', 'terminal'], params: {'debug': False, 'openai_api_key': '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', 'open_ai_api_base': 'https://api.openai.com/v1', 'proxy': '', 'request_timeout': 120, 'model_name': 'gpt-3.5-turbo', 'no_default': False, 'top_k_results': 3, 'news_api_key': '', 'bing_subscription_key': '', 'google_api_key': '', 'google_cse_id': '', 'searx_search_host': '', 'wolfram_alpha_appid': '', 'morning_news_api_key': '', 'cuda_device': 'cpu', 'think_depth': 3, 'arxiv_summary': True, 'morning_news_use_llm': False, 'llm_api_key': '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', 'llm_api_base_url': 'https://api.openai.com/v1'}
[INFO][2023-06-02 00:58:28][__init__.py:29] - [visual_dl] init failed, error_info: ModuleNotFoundError("No module named 'torch'")
[INFO][2023-06-02 00:58:28][__init__.py:29] - [visual_dl] init failed, error_info: ModuleNotFoundError("No module named 'torch'")
[INFO][2023-06-02 00:58:28][chatgpt.py:183] - proxy no find, directly request to chatgpt instead
[INFO][2023-06-02 00:58:28][chatgpt.py:183] - proxy no find, directly request to chatgpt instead
[INFO][2023-06-02 00:58:28][victorinox.py:86] - use_tools=['terminal', 'python', 'url-get', 'meteo-weather'], params: {'debug': False, 'openai_api_key': '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', 'open_ai_api_base': 'https://api.openai.com/v1', 'proxy': '', 'request_timeout': 120, 'model_name': 'gpt-3.5-turbo', 'no_default': False, 'top_k_results': 3, 'news_api_key': '', 'bing_subscription_key': '', 'google_api_key': '', 'google_cse_id': '', 'searx_search_host': '', 'wolfram_alpha_appid': '', 'morning_news_api_key': '', 'cuda_device': 'cpu', 'think_depth': 3, 'arxiv_summary': True, 'morning_news_use_llm': False, 'llm_api_key': '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', 'llm_api_base_url': 'https://api.openai.com/v1'}
[INFO][2023-06-02 01:05:18][__init__.py:29] - [visual_dl] init failed, error_info: ModuleNotFoundError("No module named 'torch'")
[INFO][2023-06-02 01:05:18][__init__.py:29] - [visual_dl] init failed, error_info: ModuleNotFoundError("No module named 'torch'")
[INFO][2023-06-02 01:05:18][chatgpt.py:183] - proxy no find, directly request to chatgpt instead
[INFO][2023-06-02 01:05:18][chatgpt.py:183] - proxy no find, directly request to chatgpt instead
[INFO][2023-06-02 01:05:18][victorinox.py:86] - use_tools=['python', 'url-get', 'meteo-weather', 'terminal'], params: {'debug': False, 'openai_api_key': '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', 'open_ai_api_base': 'https://api.openai.com/v1', 'proxy': '', 'request_timeout': 120, 'model_name': 'gpt-3.5-turbo', 'no_default': False, 'top_k_results': 3, 'news_api_key': '', 'bing_subscription_key': '', 'google_api_key': '', 'google_cse_id': '', 'searx_search_host': '', 'wolfram_alpha_appid': '', 'morning_news_api_key': '', 'cuda_device': 'cpu', 'think_depth': 3, 'arxiv_summary': True, 'morning_news_use_llm': False, 'llm_api_key': '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', 'llm_api_base_url': 'https://api.openai.com/v1'}
[INFO][2023-06-02 01:07:09][__init__.py:29] - [visual_dl] init failed, error_info: ModuleNotFoundError("No module named 'torch'")
[INFO][2023-06-02 01:07:09][__init__.py:29] - [visual_dl] init failed, error_info: ModuleNotFoundError("No module named 'torch'")
